users/kimurayoshimi/4.js

```diff
@@ -0,0 +1,3 @@
+const a = (b) => b.filter((value) => value && value % 2 === 1);
```

Comment:
Q
Why are you first confirming the existence of value?

Reply:
I had unnecessarily included conditional branch processing. I fixed it.

830e192